<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amulet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/amulet.png</normaloff>:/icons/amulet.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="DropTableView" name="queueTableView"/>
      </item>
      <item>
       <widget class="QGroupBox" name="settingsGrp">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QGridLayout" name="settingsLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="formatLbl">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>15</height>
              </size>
             </property>
             <property name="text">
              <string>Format:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="formatBox">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConvert"/>
   <addaction name="separator"/>
   <addaction name="actionAddFiles"/>
   <addaction name="actionAddDir"/>
   <addaction name="actionClearList"/>
   <addaction name="separator"/>
   <addaction name="actionProperties"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionAddFiles">
   <property name="text">
    <string>Add file(s)...</string>
   </property>
  </action>
  <action name="actionAddDir">
   <property name="text">
    <string>Add dir...</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Convert</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClearList">
   <property name="text">
    <string>ClearList</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DropTableView</class>
   <extends>QTableView</extends>
   <header>drop_table_view.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
